thinc/extra/wrappers.py

```diff
@@ -20,13 +20,14 @@ def __init__(self, model):
         self._model = model
         self._optimizer = None
 
-    def begin_update(self, x_data, drop=0.):
+    def begin_update(self, x_data, drop=0., sgd=None):
```

Comment:
Ah, this isn't quite correct: we pass the `sgd` callable in during the backward pass --- `begin_update` is the forward pass.

Reply:
Sorry! Indeed! It is the call to the method in examples/wrap_pytorch. py which was wrong then. I will fix this.